python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -410,16 +414,27 @@ def plot_one_image(ax, data, name):
             ax.plot((x_line, x_line + 1.0/scale), (y_line, y_line), color="blue", lw=6)
             ax.plot((x_line, x_line + 1.0/scale), (y_line, y_line), color="yellow", lw=2)
             ax.axis("off")
-            ax.set_title(name)
+            if name is not None:
+                ax.set_title(name)
 
         try:
-            fig, (ax1, ax2, ax3) = plt.subplots(1, 3, constrained_layout=True)
-            plot_one_image(ax1, template.image.array, "Template")
-            plot_one_image(ax2, science.image.array, "Science")
-            plot_one_image(ax3, difference.image.array, "Difference")
+            len_sizes = len(sizes)
+            fig, axs = plt.subplots(len_sizes, 3, constrained_layout=True)
```

Comment:
Is it worth using `subplots_adjust(hspace=0)` (or 0.01 or something else small) here to remove (most of?) the space between the top and bottom cutouts? Probably want some space between them, but there's quite a lot right now.

Reply:
@parejkoj About your broader concern of trying to support more than 2 cutouts, I personally think the changes I've brought has made the code more generalized. We never know if someone may require more cutouts in future- just like the original code didn't account for someone requiring cutouts of more than 1 size. I've run the code several times with different number of cutout sizes, and as such I couldn't see any performance issues.

Reply:
I tried using `subplots_adjust(hspace=0)` with a couple of other values for hspace, but something strange happened- the horizontal space between the plots actually increased instead of decreasing. The minimum space is when I do not use the adjust method at all.

Reply:
"More general" isn't always a benefit, especially regarding visualization code. "More general" means more code complexity to manage and what that unknown future user wants might not be what you implemented. It's not too bad for now, but as new features get added to the cutouts code, being too general might be a problem in the future.

Too bad about the hspace settings.